addons/hr_timesheet/controllers/portal.py

```diff
@@ -57,6 +57,8 @@ def _get_search_domain(self, search_in, search):
             search_domain = OR([search_domain, [('employee_id', 'ilike', search)]])
         if search_in in ('task', 'all'):
             search_domain = OR([search_domain, [('task_id', 'ilike', search)]])
+        if search_in == 'parent_task':
```

Comment:
```suggestion
        if search_in == 'parent_task_id':
```
Would you mind calling your search with the name of the field? It will generate fewer conflicts with my ongoing portal refactor that way